pulp_ansible/tests/functional/api/collection/v3/test_collection.py

```diff
@@ -309,27 +309,17 @@ def test_collection_download(artifact, pulp_client, collection_detail):
     assert f.content == tarball
 
 
-def test_collection_upload_repeat(pulp_client, known_collection, pulp_dist):
+def test_collection_upload_repeat(pulp_client, artifact, pulp_dist, collection_upload):
```

Comment:
what kind of a fixture is "artifact"? If it's a pulp_ansible one can we name it `collection_artifact`?

Reply:
Are the changes in this file supposed to be part of this PR at all?

Reply:
Yes, I realized the tests for collection upload were wrong and incorrectly passing when it shouldn't.